components/treelist/virtual-scrolling.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Virtual Scrolling
+page_title: TreeList - Sorting
+description: Enable and configure the virtual scrolling of rows in the TreeList for Blazor.
+slug: treelist-virtual-scrolling
+tags: telerik,blazor,treelist,virtual,rows,scrolling
+published: True
+position: 25
+---
+
+# TreeList Virtual Scrolling
+
+
+Virtual scrolling provides an alternative to paging. Instead of utilizing a pager, the user vertically scrolls through all records in the data source.
```

Comment:
```suggestion
Virtual scrolling provides an alternative to paging. Instead of utilizing a pager, the user scrolls vertically through all records in the data source.
```